cmake/common.cmake

```diff
@@ -412,19 +389,24 @@ set(CCF_NETWORK_TEST_ARGS
 )
 
 # SNIPPET_START: Lua generic application
-add_enclave_lib(luagenericenc
+add_ccf_app(luageneric
   SRCS ${CCF_DIR}/src/apps/luageneric/luageneric.cpp
 )
-sign_app_library(luagenericenc
+sign_app_library(luageneric.enclave
   ${CCF_DIR}/src/apps/luageneric/oe_sign.conf
   ${CCF_DIR}/src/apps/sample_key.pem
 )
 # SNIPPET_END: Lua generic application
 
-add_enclave_lib(jsgenericenc
+add_ccf_app(jsgeneric
   SRCS ${CCF_DIR}/src/apps/jsgeneric/jsgeneric.cpp
+  LINK_LIBS_ENCLAVE
+    quickjs.enclave
+    -lgcc
+  LINK_LIBS_VIRTUAL
+    quickjs.host
```

Comment:
quickjs needs libgcc here too, for the same reasons (maths stuff used by bigint).

Reply:
Afaict we only need the explicit `-lgcc` as a result of `-nostdlib` (or one of the similar args) - the virtual build gets these builtins automatically.